src/mopidy_spotify/web.py

```diff
@@ -9,6 +9,7 @@
 from email.utils import parsedate_to_datetime
 from enum import Enum, unique
 from http import HTTPStatus
+from threading import Lock
```

Comment:
Personally I would prefer just importing threading here, but if this better matches the current code style than leave it as is. Reasoning being that having the module in there makes it clearer which/what Lock this is.